### PR TITLE
Fix issues: 236, 275, 276 & 248

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose
Creating new flat UI design for the scholarX page

## Goals
Developing design according to the design provided

## Approach
Developing the web page in html using bootstrap

## Security checks
 Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
java 1.8, ubuntu 19.04, chrome, tomcat9

### CODE REVIEW

File: src/main/webapp/onelive.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="Start your development with a Design System for Bootstrap 4.">
+    <meta name="author" content="Creative Tim">
+    <title>SEF site</title>
+    <!-- Favicon -->
+    <link href="assets/img/brand/favicon.png" rel="icon" type="image/png">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove this if using Montserrat

File: src/main/webapp/onelive.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <link type="text/css" href="assets/css/global-custom.css" rel="stylesheet">
+
+</head>
+
+<body>
+<header class="header-global">
+    <!--nav bar goes here-->
+    <div id="navbar">
+
+    </div>
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove unwanted blank space.

```suggestion
<div id="navbar"></div>
```

File: src/main/webapp/onelive.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+</header>
+<!-- to use the dynamic navbar, use this instead of above tag => * <div id="navbar"></div> * -->
+
+<main>
+    <!--main content goes here-->
+</main>
+
+<footer class="footer">
+    <div id="footer">
+
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove unwanted blank space.

File: src/main/webapp/onelive.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+    <!--footer goes here -->
+</footer>
+
+<!-- to use the dynamic footer, use this instead of above tag => *<div id="footer"></div>* -->
+
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove unwanted blank space.

File: src/main/webapp/onelive.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+<!-- Argon JS -->
+<script src="assets/js/argon.js?v=1.0.1"></script>
+<script src="assets/js/global-custom-js.js"></script>
+<!-- Script for static navbar and footer -->
+<script src="assets/js/navigation.js"></script>
+<script>
+    $(function () {
+        loadNavAndFooter('assets/content/static')  //relative path to content directory
+        // link to the navbar, footer content goes here
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Unwanted comment line

File: src/main/webapp/scholarx.html [hunk 6]
```diff
@@ -215,59 +68,424 @@ <h6 class="heading text-primary mb-md-1">International Research
...
+                                                            class="plusMinusSign fa fa-plus float-right"></i></div>
+                                                </button>
+                                            </h5>
+                                        </div>
+                                        <div
+                                                id="collapseTwo"
+                                                class="collapse"
+                                                aria-labelledby="headingTwo"
+                                                data-parent="#accordionExample"
+                                        >
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx.html]:
```
 <div id="collapseTwo"
      class="collapse"
      aria-labelledby="headingTwo"
      data-parent="#accordionExample">
```

Comment (jayasanka-sack) [reply src/main/webapp/scholarx.html]: Change other places too

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Add screenshots

**Comment by jayasanka-sack** [comment]
#236

**Comment by Gravewalker666** [comment]
screencapture-localhost-63342-sef-site-src-main-webapp-team-html-2019-11-13-20_23_49

**Comment by Gravewalker666** [comment]
#275 
#276

**Comment by Gravewalker666** [comment]
#248

**Review by Gravewalker666 — COMMENTED** [review]
Done

**Review by Gravewalker666 — COMMENTED** [review]
Done

**Comment by jayasanka-sack** [comment]
Thanks @Gravewalker666 !

**Comment by jayasanka-sack** [comment]
#236, #275, #276 #248 #265 #251

**Pull request merged**